apps/block_scout_web/lib/block_scout_web/templates/address/_tabs.html.eex

```diff
@@ -38,12 +39,14 @@
         "data-test": "internal_transactions_tab_link",
         to: AccessHelper.get_path(@conn, :address_internal_transaction_path, :index, @address.hash)
       ) %>
-  <%= link(
-        gettext("Coin Balance History"),
-        class: "card-tab #{tab_status("coin-balances", @conn.request_path)}",
-        "data-test": "coin_balance_tab_link",
-        to: AccessHelper.get_path(@conn, :address_coin_balance_path, :index, @address.hash)
-      ) %>
+  <%= if !smart_contract_native do %>
```

Comment:
agreed, coin balance for staking contract makes sense to have it in my opinion.

Maybe that's not the case for the other contracts,.

Reply:
If condition removed.

Regarding the balance of the forger stake we just need to keep in mind that the value displayed might not be equal to the total amount staked using the `delegate` contract method because the contract can potentially receive funds also in these ways:
- forward transfer from mainchain directly to native contract address
- after a contract address destroy if the return address is set to the native contract one

Funds received with these 2 previous ways are included in the contract balance but can't be move from there.